app.test.js

```diff
@@ -127,3 +126,50 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("get all comments for an article", () => {
+    test("200: sends an array of comments belonging to a single article to the client", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.comments).toEqual(expect.any(Array));
+          expect(Object.keys(response.body.comments[0])).toEqual(
+            expect.arrayContaining([
+              "comment_id",
+              "votes",
+              "created_at",
+              "author",
+              "body",
+              "article_id"
+            ])
+          );
```

Comment:
I would say just checking the first result has the keys you expect doesn't really prove that all of the articles have the keys you expect also. We also aren't checking what the values are or what data type they should be.

Consider trying to use a forEach here to look at each article and then using something like hasOwnProperty 😄

Something along the lines of
articles.forEach((article) => {
expect(article).hasOwnProperty("article_id", expect.any(Number))

etc. etc. for each property you expect to find 😄